File: src/main/java/bot/util/settings/SettingsReceiver.java
package bot.util.settings;

import bot.run.AbitVTBot;

public class SettingsReceiver {
    private final AbitVTBot bot;

    public SettingsReceiver(AbitVTBot bot) {
        this.bot = bot;
    }

    public void setSettings(SettingsInvoker invoker) {
        invoker.add("⚙️ Settings \uD83D\uDEE0", new Settings(bot));
        invoker.add("⚙️ En \uD83C\uDDFA\uD83C\uDDF8", new EN(bot));
        invoker.add("⚙️ Ru \uD83C\uDDF7\uD83C\uDDFA", new RU(bot));
    }
}

File: src/main/java/bot/util/settings/SettingsReceiver.kt
package bot.util.settings

import bot.run.AbitVTBot

class SettingsReceiver(private val bot: AbitVTBot) {

    fun setSettings(invoker: SettingsInvoker) {
        val settings: Settings = Settings(bot)
        invoker.add("⚙️ Settings \uD83D\uDEE0", settings)
        invoker.add("⚙️ Настройки \uD83D\uDEE0", settings)
        invoker.add("⚙️ En \uD83C\uDDFA\uD83C\uDDF8", EN(bot))
        invoker.add("⚙️ Ru \uD83C\uDDF7\uD83C\uDDFA", RU(bot))
    }
}